pkg/apis/policy/v1alpha1/register.go

```diff
@@ -41,6 +42,7 @@ func addKnownTypes(scheme *runtime.Scheme) {
 	scheme.AddKnownTypes(SchemeGroupVersion,
 		&PodDisruptionBudget{},
 		&PodDisruptionBudgetList{},
+		&v1.ListOptions{},
```

Comment:
Wow, that code is wrong.

On Tue, Jun 21, 2016 at 12:18 AM, Chao Xu <notifications@github.com> wrote:

> In pkg/apis/policy/v1alpha1/register.go
> <https://github.com/kubernetes/kubernetes/pull/27733#discussion_r67805956>
> :
>
> > @@ -41,6 +42,7 @@ func addKnownTypes(scheme *runtime.Scheme) {
> >  	scheme.AddKnownTypes(SchemeGroupVersion,
> >  		&PodDisruptionBudget{},
> >  		&PodDisruptionBudgetList{},
> > +		&v1.ListOptions{},
>
> DeleteOptions makes sense. I'm not sure about ExportOptions. It's always
> treated as "v1":
> https://github.com/kubernetes/kubernetes/blob/master/pkg/apiserver/resthandler.go#L135,
> though that code might be wrong.
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/kubernetes/kubernetes/pull/27733/files/1c74f17935c2b2cd29db72eda4f7a271b9d3bf24#r67805956>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/ABG_p3syUQTAoe1yPQ_6qu4t5C1959qIks5qN2YNgaJpZM4I6N5r>
> .
>